public/achievementinspector.php

```diff
@@ -325,7 +331,7 @@ function toggleAllCodeRows() {
                 echo "<a class='btn w-full flex justify-center py-2' href='/achievementinspector.php?g=$gameID&f=5'>Unofficial Achievement Inspector</a>";
             }
 
-            if ($fullModifyOK) {
+            if ($fullModifyOK || $partialModifyOK) {
```

Comment:
This is already within a `$fullModifyOK || $partialModifyOK` conditional. Can this conditional be removed?

Reply:
Yes, these have now been removed, functionality remains the same